hadoop-hdds/docs/content/_index.zh.md

```diff
@@ -20,5 +20,18 @@ weight: -10
   limitations under the License.
 -->
 
+# Apache Hadoop Ozone
 
-Ozone page translation.
+<img src="ozone-usage.png" style="max-width: 60%;"/>
```

Comment:
It seems using Hugo shortcode instead of direct HTML tag lets Hugo relativize the `img` link:

```suggestion
{{<figure src="/ozone-usage.png" width="60%">}}
```

Please also update `_index.md` with the same.

Reply:
I just tried this and it works, but the do we need to update _index.md ? since it doesn't have this probelm.

Reply:
Updating `_index.md` is beneficial even if the image works: future translations can start with it, instead of having to copy this snippet from the `.zh` version.

Also, newer Hugo seems to omit raw HTML by default.

Reply:
So convincing, I will update them in a while.